examples/loops-big-array.yaml

```diff
@@ -0,0 +1,47 @@
+# Output parameters can't be larger than 256kb. To loop over larger arrays, use an artifact.
```

Comment:
Could you add more details to what the problem is and what it's trying to solve here? Adding a link to the StackOverflow post would be great as well. I'm thinking about the people that stumble onto this example with no context.

Reply:
Great points! Added some more details.